w11_project_3/README.md

```diff
@@ -0,0 +1,89 @@
+# ![](https://ga-dash.s3.amazonaws.com/production/assets/logo-9f88ae6c9c3871690e33280fcf557f33.png) WDI Project 3 - Portfolio Project
+
+## Overview
+
+**You’ve come a long way, and it's time to show it.** This will be your most advanced project to date, and it'll hopefully be the shining beacon of bragging rights for your portfolio as you make your way out into the world.
+
+**You get to call the shots and invent your own idea**, choosing a framework & tools that are appropriate for what you want to build. Use everything you've learned so far, and **tackle something that'll push you a little outside of your comfort zone.**
+
+Make sure it's **something you can accomplish in the limited time we have**, and make sure it's **something that'll be impressive visually**.
+
+We expect most students will complete this project individually, but pairs may be approved on a case-by-case basis. See the planning deliverables section below for extra planning requirements for pairs.
+
+## Big Goals
+
+* **Discover or invent an idea**, preferably one with some group of users that would jump at the chance to really use your product.
+* **Design a product _you_ want to build**, and choose appropriate technologies to build it.
+* **Pitch your idea to your instructors** and incorporate their feedback.
+* **Make productive use of your time**, and balance responsibilities to make a complete, impressive-looking project.
+* **Focus on writing solid, real-world code** that doesn’t repeat itself and is well-documented.
+
+## Core Technical Requirements
+
+**Your app must:**
+
+* **Use a database**, whether that's one we've covered in class or one you want to learn.
+* **Create a complete product**, which most likely means multiple related resources and CRUD functionality for each.
+* **Build a full-stack application** by making your own backend and your own front-end.
+* Up your design & style game; **make a product that's impressive looking** to kick your portfolio up a notch.
+* **Deploy your application online** so it's publically accessible.
+* **Create a focused product**. Know which features are essential to build for your MVP and which to set aside for later.
+* **Craft thoughtful user stories** that are significant enough to help you ensure a pleasing and logical user experience.
+* **Handle errors gracefully**, and provide useful feedback to users when errors or validation failures do occur.
+
+## Further Challenge Ideas
+
+* **Incorporate external APIs** to add data and functionality to your application.
+* **Use a data visualization library** to help users understand your core data.
+* **Use oAuth logins** to allow users to access data from other services.
+* **Build something you can really launch** and recruit a base of actual users.
+* **Test** critical components of your code to ensure it works.
```

Comment:
Should this be a core technical requirement instead? I don't think anyone will do it if it's listed under further challenges.